slides/slides.md

```diff
@@ -55,25 +55,28 @@ class: dense
   - 1-liner setup
   - Expo Go: great for POCs 🚀
   - Limited access to native code and APIs
-  - Can be hard to update 🤯
+  - Easier updates
```

Comment:
@robwalkerco I did have a question on this: I've seen people struggle to update expo apps too because all of the dependency versions need to be set to certain versions.

Maybe we need to capture this point differently? What do you think?

Reply:
haha. Perhaps there are just different difficulties in updating then!
With RN, the difficulties come from having to resolve all the changes to the native files, which is not a problem with Expo